### DISCUSSION TITLE
Table Checkbox Selection

### DISCUSSION THREAD

**Discussion by heath-capstone** [primefaces/community#965] [Figma UI Kit] [answered]
Hello all! I'm just getting started with PrimeOne 2.2.0, as my development team is using PrimeNG and Prime React.

We have existing functionality where the first column in the table is a checkbox; while exploring the PrimeOne library I noticed that there doesn't appear to be an option to have the table cell be a checkbox or radio button. I double checked the documentation for AngularNG and including a checkbox or radio button is included as a feature (see attached image). 

It's totally possible that I just missed or misunderstood the functionality, if so sorry about that! If, however, there isn't an option to include checkboxes/radio buttons into table cells I would like to officially request that functionality! Thanks! 

![figma_table_checkbox_selection](https://github.com/primefaces/community/assets/156356168/dad7608c-33ee-4660-9443-0859c5c35559)

**Comment by onursenture** [comment, marked as answer]
Hello, 

We have UI component libraries with various features for different components. This makes adding all features to Figma a challenging process. As a result, our Figma UI Kit may need some features of certain components. We are attempting to address these gaps based on the feedback we receive.

As a design team, we have noted the shortcomings you identified. We will consider them for improvements in future versions.

Thank you! 🙏

**Reply by heath-capstone** [reply]
Thanks @onursenture! Any idea on a timeline for when table selection might be available? Just trying to do some impact/effort calculation on whether I should try to roll a custom component to provide this functionality. Thanks! 👍

**Reply by onursenture** [reply]
The likelihood of us doing an interim release before updating for the new token is relatively low, so it might be better for you to proceed with creating your custom component instead of waiting.
